public/app/percona/backup/components/BackupInventory/RestoreBackupModal/RestoreBackupModal.utils.ts

```diff
@@ -0,0 +1,11 @@
+import { DATABASE_LABELS } from 'app/percona/shared/core';
+import { Backup } from '../BackupInventory.types';
+import { formatDataModel } from '../BackupInventory.utils';
+import { RestoreBackupFormProps, ServiceTypeSelect } from './RestoreBackupModal.types';
+
+export const toFormProps = ({ vendor, serviceId, serviceName, dataModel }: Backup): RestoreBackupFormProps => ({
```

Comment:
Let's use one interface for both params and return value

Reply:
Not sure if I follow. You mean having a single interface instead of `Backup` and `RestoreBackupFormProps` ?
They are different, as the form has a service type field ("SAME" vs "COMPATIBLE") that doesn't mean anything on the outside, for example. 
Also, the form takes a `SelectableValue` with `{ label: serviceName, value: serviceId }` for the service field, which allows to extract the service ID right away when submitting it. That field is not relevant outside of the form as well.

Reply:
```typescript
type ToFormProps = (props: Backup) => RestoreBackupFormProps;
```

```suggestion
export const toFormProps: ToFormProps = ({ vendor, serviceId, serviceName, dataModel }) => ({
```

something like this

Reply:
got it 👍